doc/getting_started/index.rst

```diff
@@ -346,23 +346,32 @@ including: compiler, assembler, linker, and their dependencies.
             chmod +x zephyr-sdk-0.11.2-setup.run
             ./zephyr-sdk-0.11.2-setup.run -- -d ~/zephyr-sdk-0.11.2
 
-      #. Set environment variables to let the build system know where to
-         find the toolchain programs:
+         .. note::
+	    It is recommended to install the Zephyr SDK at one of the following locations:
 
-         .. code-block:: bash
+            * ``$HOME/zephyr-sdk[-x.y.z]``
+            * ``$HOME/.local/zephyr-sdk[-x.y.z]``
+            * ``$HOME/.local/opt/zephyr-sdk[-x.y.z]``
+            * ``/opt/zephyr-sdk[-x.y.z]``
+            * ``/usr/zephyr-sdk[-x.y.z]``
+            * ``/usr/local/zephyr-sdk[-x.y.z]``
+
+	    where ``[-x.y.z]`` is optional text, and can be any text, for example ``-0.11.2``.
```

Comment:
Add a note that links to the extended info when you choose to install in a different location

Reply:
yep, good point.

Reply:
done.